dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/EmbeddedServerImpl.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2012, 2016 IBM Corporation and others.
+ * Copyright (c) 2012, 2016, 2019 IBM Corporation and others.
```

Comment:
Thanks for remembering to update the copyright!  For changes to parts, we keep the first and the last copyright date.  So this should be `2012, 2019` instead of `2012, 2016, 2019`.  This applies to all of the copyrights you changed here.